docs/tutorials/complementary_color_prediction.md

```diff
@@ -0,0 +1,194 @@
+# Predicting Complementary Colors with deeplearn.js
```

Comment:
Keep this file in demos/complementary-color-prediction/ - doesn't have to be in docs/tutorials. The website will pick it up and show it on the right side in https://pair-code.github.io/deeplearnjs/docs/tutorials/index.html automatically